docs/sdks/debugging.mdx

```diff
@@ -57,6 +57,22 @@ For server SDKs, these reasons for the value you are seeing can be:
 - `Unrecognized`: the sdk was initialized, but this config did not exist in the set of values
 - `Uninitialized`: the sdk was not yet successfully initialized
 
+In newer server SDKs, the reasons will be a combination of initialization source and evaluation reason:
```

Comment:
Mmmm. At first I wanted to combine all of this but I see why its separate.  Can you make a heading for the client and server sdk sections?

So

#### Client SDK Reasons
<blah>

#### Server SDK Reasons
<blah>